grafana-plugin/src/containers/AlertReceiveChannelCard/AlertReceiveChannelCard.tsx

```diff
@@ -63,6 +65,22 @@ const AlertReceiveChannelCard = observer((props: AlertReceiveChannelCardProps) =
             <Text type="primary" size="medium">
               <Emoji className={cx('title')} text={alertReceiveChannel.verbal_name} />
             </Text>
+            {store.hasFeature(AppFeature.Webhooks2) && (
+              <CopyToClipboard text={alertReceiveChannel.id}>
+                <IconButton
+                  variant="primary"
+                  tooltip={
+                    <div>
+                      ID {alertReceiveChannel.id}
+                      <br />
+                      (click to copy ID to clipboard)
+                    </div>
+                  }
+                  tooltipPlacement="top"
+                  name="info-circle"
+                />
+              </CopyToClipboard>
+            )}
```

Comment:
_nit_; might make sense to move this series of components to a shared component that is reused in these 3 cases in the PR